server.js

```diff
@@ -4734,6 +4734,52 @@ cache(function(data, match, sendBadge, request) {
   }
 }));
 
+// Arch user repository (AUR) integration.
+camp.route(/^\/aur\/(version|votes|license)\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+    var info = match[1];
+    var package = match[2];
+    var format = match[3];
+    var apiUrl = 'https://aur.archlinux.org/rpc.php?type=info&arg=' + package;
+    var badgeData = getBadgeData('AUR', data);
+    request(apiUrl, function(err, res, buffer) {
+        if (err != null) {
+            badgeData.text[1] = 'inaccessible';
+            sendBadge(format, badgeData);
+            return;
+        }
+        try {
+            var data = JSON.parse(buffer).results;
+            if (info === 'version') {
+                var vdata = versionColor(data.Version);
+                badgeData.text[1] = vdata.version;
+                if (data.OutOfDate === null) {
+                    badgeData.colorscheme = 'blue';
+                } else {
+                    badgeData.colorscheme = 'orange';
+                }
+                sendBadge(format, badgeData);
+            } else if (info === 'votes') {
+                var votes = data.NumVotes;
+                badgeData.text[1] = votes;
```

Comment:
Do you want to have the left-hand side part of the badge say "votes", to be descriptive?